### PR TITLE
feat: add fading Lottie

### PR DESCRIPTION
**Pull request by vatana7**
# Description

I've seen that on initial load of the page, Lottie seem to pop up suddenly so I made some adjustment for a fade in effect for Lottie to increase better UI.

## Type of change

- [x ] New feature (non-breaking change which adds functionality)

==Before==
before

==After==
after

### CODE REVIEW

**Comment by kartikcho** [comment]
@vatana7 could you upload a full video (I think you can upload videos now) of the change in action? The gifs aren't clear and layout in the 2nd one looks broken.

Ideally the vercel deployment would've helped out but waiting on @saadpasta to fix #540

**Comment by vatana7** [comment]
I don't have any access to screen recorder at the moment but if you're curious I made some change with the div that cover the lottie to add in a little fade in effect on initial load to look better overall. If you think it's helpful please merge the code 🤟🏻. 
Also after #540 my effect would still be relevant as well

File: src/containers/greeting/Greeting.js [hunk 1]
```diff
@@ -52,7 +52,9 @@ export default function Greeting() {
           </div>
           <div className="greeting-image-div">
             {illustration.animated ? (
-              <DisplayLottie animationData={landingPerson} />
+              <div className="fade-in">
```
Comment (kartikcho) [inline src/containers/greeting/Greeting.js:55]: We could use `<Fade bottom duration={1000} distance="40px">` from line 19 so both the text and the lottie appears with the same effect together

Comment (kartikcho) [reply src/containers/greeting/Greeting.js:55]: Could you check how that looks and if they fade in from bottom together?

Comment (kartikcho) [reply src/containers/greeting/Greeting.js:55]: A new commit would also trigger a redeploy

Comment (vatana7) [reply src/containers/greeting/Greeting.js:55]: Sure! I'll take a look and get back to you!

**Comment by vatana7** [comment]
```javascript
<Fade bottom duration={5500} distance="60px">
       <DisplayLottie animationData={landingPerson} />
</Fade>
```
This is the adjustment I made as you suggested. Please have a look @kartikcho 

Kapture 2022-10-18 at 09 59 21

**Comment by saadpasta** [comment]
@vatana7 Can we make the text and image show up at the same time currently the image slides in after the text

**Comment by abdullahnasir4** [comment]
@saadpasta Can you share the url for landingperson.json from lottiefiles.

**Pull request closed without merging**